userspace/libsinsp/plugin.cpp

```diff
@@ -1047,4 +1068,87 @@ bool sinsp_extractor_plugin::resolve_dylib_symbols(void *handle, std::string &er
 	return true;
 }
 
+sinsp_capture_plugin::sinsp_capture_plugin()
+{
+	memset(&m_capture_plugin_info, 0, sizeof(m_capture_plugin_info));
+}
+
+sinsp_capture_plugin::~sinsp_capture_plugin()
+{
+	close();
+	destroy();
+}
+
+capture_plugin_info *sinsp_capture_plugin::plugin_info()
+{
+	return &m_capture_plugin_info;
+}
+
+bool sinsp_capture_plugin::open(const char *param, ss_plugin_rc &rc)
+{
+	ss_plugin_rc orc;
+
+	if(!plugin_state())
+	{
+		return false;
+	}
+
+	m_capture_plugin_info.handle = m_capture_plugin_info.open(plugin_state(), param, &orc);
+
+	rc = orc;
```

Comment:
```suggestion
	m_capture_plugin_info.handle = m_capture_plugin_info.open(plugin_state(), param, &rc);
```

(and delete the `orc` variable)

Reply:
Thanks!